resources/server/src/landingpage/osv-scanner.toml

```diff
@@ -0,0 +1,23 @@
+[[IgnoredVulns]]
+id = "GHSA-8jmw-wjr8-2x66"
+reason = "...the JS application is not built nor shipped anymore... Look to https://github.com/qgis/QGIS/pull/55748"
+
+[[IgnoredVulns]]
+id = "GHSA-pfrx-2q88-qq97"
+reason = "...the JS application is not built nor shipped anymore... Look to https://github.com/qgis/QGIS/pull/55748"
```

Comment:
The correct reason to ignore is that the built application is meant to be run on the client and not on the server.

Reply:
Resolved in 7a97ab7793f031d6a94586eefe54702b4ed35dcb